csv/src/main/java/com/fasterxml/jackson/dataformat/csv/CsvMapper.java

```diff
@@ -460,6 +487,18 @@ protected void _addSchemaProperties(SerializerProvider ctxt, CsvSchema.Builder b
         BeanDescription beanDesc = ctxt.introspectBeanDescription(pojoType);
         final AnnotationIntrospector intr = ctxt.getAnnotationIntrospector();
         for (BeanPropertyDefinition prop : beanDesc.findProperties()) {
+            if (view != null) {
+                boolean viewVisible = false;
+                for(Class<?> propView : prop.findViews()) {
```

Comment:
Needs null check, `findViews()` can return `null`. Also note that in databind `BeanDeserializerFactory` method `addBeanProps()`, there is:

```
                Class<?>[] views = propDef.findViews();
                if (views == null) {
                    views = beanDesc.findDefaultViews();
                }
```

Reply:
Thanks, updated.